client/landing/gutenboarding/available-designs-config.json

```diff
@@ -11,7 +11,8 @@
 			},
 			"categories": [ "featured" ],
 			"is_premium": false,
-			"is_fse": true
+			"is_fse": true,
+			"is_anchorfm": false
```

Comment:
Do we need to add this prop there or is it to be more explicit?

Reply:
There's no `is_fse: false` neither in many of the designs.

Maybe these would be better as an array `flows: [ 'fse', 'anchor', ... ]` ?

Reply:
To elaborate: actually, instead of thinking of these as "flows", it might be better to think of these as "supported features". Imagine we'd have adjustments in the flow for blog, e-commerce, business address, podcasting, or any combination of those.

Reply:
I generally prefer being explicit over not.  For example, the current implementation of optional parameter `is_anchorfm` leads to this expression: 

`isAnchorFmSignup === ( design.is_anchorfm ?? false )`.

I'd prefer `isAnchorFmSignup == design.is_anchorfm`, but lint rules guide me towards using triple equals, and `false === undefined` isn't true.  It's easier to only have to think about one possible value (`false`) rather than two (`false` or `undefined`).

Reply:
Created PR:  _Move is_anchor attribute on gutenboard designs to feature array #48062_

Note this only concerns itself with the anchor feature for now, to limit scope.  We can add other features after this is merged.

Reply:
My only concern is maintainability -- every new design requires this prop to be added. I am generally on the side of things being more explicit. My comment is not a blocker by any means!

Reply:
Please see PR #48062

Reply:
@mreishus while being explicit, will this then fail loudly if someone doesn't add the key in all designs as well _doesn't_ manually test Anchor flows? Note also that some day these hopefully will come from an API.

Reply:
Oh and my primary concerns was situation where we have dozen `is_` things here and array felt like more scalable when we already/only have 3. :-)

Definetely not a blocker from my perspective tho 👍

Reply:
* The `is_anchor` key is still optional, I only added extra false values to be explicit.
* I am not quite understanding the second point: why a new design without anchor set would require manual testing of Anchor flows.  I don't believe it should.
* PR #48602 removes the `is_anchor` attribute for a new features array.